cluster/manifests/aws-node-decommissioner/aws-iam-role.yaml

```diff
@@ -0,0 +1,29 @@
+Metadata:
```

Comment:
This must be defined as part of the `cluster.yaml` CF stack look there for examples.